automation/terraform/monitoring/README.md

```diff
@@ -0,0 +1,180 @@
+``` 
+
+##### Check alerts against recommended [best practices](https://prometheus.io/docs/practices/rules/)
+
+###### automation
+
+Executed by CI's *Lint/TestnetAlerts* [job](https://github.com/MinaProtocol/mina/blob/develop/buildkite/src/Jobs/Lint/TestnetAlerts.dhall) when a change is detected to the testnet-alerts template file.
+
+###### manual steps
+
+```
+    terraform apply -target module.o1testnet_alerts.null_resource.alert_rules_check
+```
+
+#### Deployment
+
+Deploying testnet alert rules entails syncing the rendered configuration in source with the Prometheus instance rules config found [here](https://o1testnet.grafana.net/a/grafana-alerting-ui-app/?tab=rules&rulessource=grafanacloud-o1testnet-prom). Appropriate AWS access is necessary for authenticating with Grafanacloud and should be similar to, if not the same, as those used for deploying testnets.
+
+###### automation
+
+Executed by CI's *Release/TestnetAlerts* [job](https://github.com/MinaProtocol/mina/blob/develop/buildkite/src/Jobs/Release/TestnetAlerts.dhall) when a change is detected to the testnet-alerts template file and linting/checking of alerts has succeeded.
+
+###### manual steps
+
+```
+    terraform apply -target module.o1testnet_alerts.docker_container.sync_alert_rules
+```
+
+**Note:** operation will sync provisioned alerts with exact match of alert file state (e.g. alerts removed from the alert file will be unprovisioned on Grafanacloud)
+
+**Warning:** Unfortunately terraform silences errors which cortextool is producing, for example when syncing rules. In order to debug sync rules process:
+- Obtain secrets (id,address,key) from aws secret manager 
+- Run above terraform command once again
+- Run below command in automation/terraform/monitoring folder: 
+
+```
+    ~/.local/bin/cortextool rules sync --rule-files alert_rules.yml --id ... --address ... --key ... 
+```
+
+## Alert Status
+
+#### GrafanaCloud Config
+
+To view the current testnet alerting rules config or verify that changes were applied correctly following a deployment, visit this Grafanacloud rules config [site](https://o1testnet.grafana.net/alerting/list?search=datasource:grafanacloud-o1testnet-prom).
+
+**Note:** ensure the datasource is set to `grafanacloud-o1testnet-prom` to access the appropriate ruleset. 
+
+#### Alertmanager UI
+
+Alerting rule violations can also be viewed in Grafanacloud's Alertmanager [UI](https://alertmanager-us-central1.grafana.net/alertmanager/#/alerts). This site provides an overview of all violating rule conditions in addition to rules that have been silenced.
+
+#### PagerDuty
+
+PagerDuty is O(1) Lab's primary alert receiver and is currently configured with a single [service](https://o1labs.pagerduty.com/service-directory/PY2JUNP) for monitoring Mina testnet deployments. This service receives alert notifications requiring attention from the development team to assist in repairing issues and restoring network health.
```

Comment:
```suggestion
PagerDuty is O(1) Lab's primary alert receiver and is configured with a single [service](https://o1labs.pagerduty.com/service-directory/PY2JUNP) for monitoring Mina testnet deployments. This service receives alert notifications requiring attention from the development team to assist in repairing issues and restoring network health.
```

Reply:
everything is "current" so let's exclude it
see https://developers.google.com/style/word-list#currently